### PR TITLE
Nd desktop styling

### LINKED ISSUE #102: Add search bar functionality (prepare for working endpoint)

### LINKED ISSUE #103: Tablet Styling

### LINKED ISSUE #125: Create a paw_print.ico for window tab
- Create a favicon with our pawprint logo to display on the tab in a desktop view

### PR DESCRIPTION
**Pull request by npdarrington**
### Participating Group Members:

- Nathan Darrington

### Code Highlights:

- Adjust styling to fit all views, mobile, tablet, and desktop.
- Adds new assets that were stretching too far scaling to a bigger size.
- Incorporates all current changes from the `main` branch inside of this branch and adjust the styling to correctly match styling.

### Have Tests Been Added?

- [x] No.
- [ ] Yes, but all tests are not passing.
- [ ] Yes, and all are passing.

### Any background context you want to provide?

N/A

### Message/Questions for reviewer:

N/A

### Issues:

- Addresses #
- Closes #125, Closes #103, Closes #102

### Screenshots (if appropriate):

Screen Shot 2021-01-13 at 2 20 07 PM
Screen Shot 2021-01-13 at 2 20 34 PM
Screen Shot 2021-01-13 at 2 20 46 PM
Screen Shot 2021-01-13 at 2 21 21 PM

### Tracking Consistency:

- [x] added appropriate labels
- [x] My code follows the code style of this project and has removed all unnecessary annotations
- [x] I have added comments on my pull request, particularly in hard-to-understand areas
- [x] All new and existing tests passed

- [x] looked at PR preview to check spelling, syntax, formatting, and completion

### CODE REVIEW

File: src/components/ResultsListItem.vue
Comment (bretm9) [inline src/components/ResultsListItem.vue:20]: Thanks for keeping ARIA accessibility in mind!

File: src/components/ResultsListItem.vue
Comment (bretm9) [inline src/components/ResultsListItem.vue:100]: These media queries are making things look much better on desktop. Thanks for all your work on this

File: src/components/ResultsMap.vue
Comment (bretm9) [inline src/components/ResultsMap.vue:34]: Did we end up deciding to put this back underneath the map? I'm good either way.

**Review by bretm9 — APPROVED** [review]
This really improves the user experience on desktop. Thanks for getting this clean look working. The map looks especially awesome on widescreen displays.

File: src/components/ResultsMap.vue
Comment (stacyp2006) [reply src/components/ResultsMap.vue:34]: Yeah, when Nathan and I were running through the site, it looked really awkward at the top, so we moved it.

File: src/components/FindPark.vue
Comment (stacyp2006) [inline src/components/FindPark.vue:35]: Oh, I've not used `aria-describedby` before. I'll have to check it out. Glad we have these labels for screen readers.

File: src/components/ResultsMap.vue
Comment (stacyp2006) [inline src/components/ResultsMap.vue:16]: Thanks for the work on the map appearance. The removal of some of the controls really improves appearance without sacrificing UX.

File: src/views/TheHome.vue
Comment (stacyp2006) [inline src/views/TheHome.vue:5]: There is nothing I don't love about this picture.

**Review by stacyp2006 — APPROVED** [review]
Thank you, thank you for all your hard work on the styling, Nathan. Really appreciate it.

File: src/components/FindPark.vue
Comment (nicolegooden) [reply src/components/FindPark.vue:35]: Same here! I'm excited to look into this more in the future.  Thanks for considering this, Nathan.

File: src/components/FindPark.vue
Comment (nicolegooden) [inline src/components/FindPark.vue:26]: For lines 26 and 34, could you change the value for `aria-describedby` so that it says something like this:

`Enter city and state to search for dog parks in that area.`

Could you make the same type of change on 34 so that `address` is replaced by `state`?

File: src/components/ResultsMap.vue
Comment (nicolegooden) [reply src/components/ResultsMap.vue:16]: Yes, I agree with Stacy! When you showed us these changes locally, I was impressed by how you were able to sift through which controls were necessary to maintain good UX.

File: src/components/ResultsMap.vue
Comment (nicolegooden) [reply src/components/ResultsMap.vue:34]: Good to know! I guess sometimes we don't know what will look good until we get closer to the end of development.

File: src/views/TheHome.vue
Comment (nicolegooden) [reply src/views/TheHome.vue:5]: Should we change this `alt` on line 5 so that it better reflects the new picture? Perhaps, `alt="picture of two dogs playing in the grass"` ?

Comment (nicolegooden) [reply src/views/TheHome.vue:5]: Also, thank you for saving that picture in our repository so that we don't have to rely on the internet keeping that picture alive (it also looks cleaner).

**Review by nicolegooden — CHANGES_REQUESTED** [review]
Nathan, I appreciate all of these design and styling changes to account for our desktop layout.  I left some comments for some changes that could be made if you agree that they are valid changes to be made.

File: src/components/FindPark.vue
Comment (npdarrington) [reply src/components/FindPark.vue:35]: My pleasure! Tried to make it usable and accessible for all users with as many disabilities as possible.

File: src/views/TheHome.vue
Comment (npdarrington) [reply src/views/TheHome.vue:5]: Good catch, I will remember to do this and re-committing the update before it is merged.

File: src/components/FindPark.vue
Comment (npdarrington) [reply src/components/FindPark.vue:26]: I will make this change as well. This was just filler text based on which updates I thought were coming. Now that we have the branches all merged together, we can better reflect the exact phrasing for what the App is capable of handling.

**Comment by npdarrington** [comment]
The changes reflecting how to search with the input, as well as the alt text for TheHome component picture has been updated.

**Review by nicolegooden — APPROVED** [review]
I appreciate your effort on this branch! I will merge this after we present at our project evaluation.

**Pull request merged**
